pymongo/common.py

```diff
@@ -17,6 +17,7 @@
 
 import collections
 import warnings
+import datetime
```

Comment:
This is a minor nit: we try to keep the imports alphabetical in pymongo. Could you move `import datetime` above `import warnings`?